Src/DependencyCollector/Shared/DependencyTrackingTelemetryModule.cs

```diff
@@ -173,11 +185,11 @@ protected virtual void Dispose(bool disposing)
             {
                 if (disposing)
                 {
-#if !NET40 && !NETCORE
+#if NET45
```

Comment:
Is it equivalent ("if NET45" and "if NET40 && !NETCORE")? `httpDesktopDiagnosticSourceListener` seems to be defined under the second condition but released under the fist condition.